src/components/ssc-exercise/index.tsx

```diff
@@ -29,67 +38,77 @@ const SSCExercise: React.FC<SSCExerciseProps> = ({chapter, steps}) => {
-      if (!completedChapters.includes(chapter)) {
-        completedChapters.push(chapter);
-        localStorage.setItem('chapterDone', JSON.stringify(completedChapters));
-      }
+  if (!steps) {
+    return <div>Loading...</div>;
+  }
 
-      router.push(
-        areAllChaptersComplete(completedChapters) ? '/ssc/feedback' : '/ssc/accomplishment',
-      );
-    } else {
-      setCurrentStep((prev) => prev + 1);
+  useEffect(() => {
+    if (!api) {
+      return;
     }
-  };
 
-  const handlePreviousStep = () => {
-    if (currentStep === 0) {
-      router.back();
-    } else {
-      setCurrentStep((prev) => prev - 1);
-    }
-  };
+    setCurrentStep(api.selectedScrollSnap());
 
-  if (!steps) {
-    return <div>Loading...</div>;
-  }
+    api.on('select', () => {
+      setCurrentStep(api.selectedScrollSnap());
+    });
+  }, [api]);
 
-  const currentStepData = steps[currentStep];
+  const handleComplete = () => {
+    console.log('complete');
+    const completedChapters = JSON.parse(localStorage.getItem('chapterDone') || '[]');
+    if (!completedChapters.includes(chapter)) {
+      completedChapters.push(chapter);
+      localStorage.setItem('chapterDone', JSON.stringify(completedChapters));
+    }
+  };
 
   return (
-    <PageLayout backgroundColor={sscMock[chapter as keyof typeof chapterMap].backgroundColor}>
-      <article className="flex h-40 flex-col items-center justify-between">
-        <HeaderWrapper
-          title={currentStepData.title}
-          handleBack={handlePreviousStep}
-          currentStep={currentStep}
-        />
-        <StepCounter currentStep={currentStep} length={steps.length - 1} />
-      </article>
-      <article className="flex flex-1 flex-col items-center justify-evenly">
-        {chapterSteps[currentStep]?.sticker && (
-          <RiveAnimation
-            key={chapterSteps[currentStep].sticker}
-            src={chapterSteps[currentStep].sticker}
-            width={300}
-          />
-        )}
-        <DescriptionWrapper>{currentStepData.description}</DescriptionWrapper>
-        {chapterSteps[currentStep]?.timer && <Timer seconds={chapterSteps[currentStep].timer} />}
-        <footer className="mt-8">
-          <Button variant="pink" onClick={handleNextStep}>
-            {currentStepData.button} <ArrowRight size={28} />
+    <PageLayout
+      backgroundColor="bg-blue"
+      header={
+        <Header>
+          <CarouselPagination steps={steps} currentStep={currentStep} />
+        </Header>
+      }
+      footer={
+        currentStep === steps.length - 1 ? (
+          <Button variant="blue" className="mx-auto" asChild onClick={handleComplete}>
+            <Link href={`/ssc/accomplishment`}>
+              {t('completeButton')} <Complete stroke="white" />
+            </Link>
+          </Button>
+        ) : (
+          <Button variant="yellow" className="mx-auto" asChild>
+            <Link href="/">{t('homeButton')}</Link>
           </Button>
```

Comment:
This button should not be a go to home button. It should scoll to next step. This can be achived with  api.scrollNext :)